widgets/matplotlib/measurement/energy_spectrum.py

```diff
@@ -53,11 +53,17 @@
 from widgets.matplotlib.base import MatplotlibWidget
 
 
+MEASURED_SUM_SPECTRUM_LINE_STYLE = "dashed"
+SIMULATED_SUM_SPECTRUM_LINE_STYLE = "dotted"
+SUM_SPECTRUM_LINE_WIDTH = 2
+Z_ORDER = 100
+
+
 class MatplotlibEnergySpectrumWidget(MatplotlibWidget):
     """Energy spectrum widget
     """
     # By default, draw spectra lines with a solid line
-    default_linestyle = "-"
+    DEFAULT_LINE_STYLE = "-"
```

Comment:
Matplotlib spells this as [linestyle](https://matplotlib.org/stable/gallery/lines_bars_and_markers/linestyles.html)

```suggestion
    DEFAULT_LINESTYLE = "-"
```

Reply:
Matplotlib spells it wrong :D